src/main/java/org/openstreetmap/atlas/geography/MultiPolygon.java

```diff
@@ -238,6 +240,55 @@ public boolean fullyGeometricallyEncloses(final PolyLine polyLine)
         return false;
     }
 
+    /**
+     * Tests to see if entire surface of the provided {@link MultiPolygon} lies within this
+     * {@link MultiPolygon}
+     * 
+     * @param that
+     *            the provided {@link MultiPolygon} to test
+     * @return true if the conditions are met, false otherwise
+     */
+    @Override
+    public boolean fullyGeometricallyEncloses(final MultiPolygon that)
+    {
+        final QuadTree<Polygon> thisOuters = QuadTree.forLocated(this.outers());
+        // each outer of that must fit within one of this' outer/inner groups
+        for (final Polygon thatOuter : that.outers())
+        {
+            boolean cleared = false;
+            for (final Polygon thisOuter : thisOuters.get(thatOuter.bounds(), thatOuter::overlaps))
+            {
+                if (thisOuter.fullyGeometricallyEncloses(thatOuter))
+                {
+                    cleared = this.getOuterToInners().get(thisOuter).stream()
+                            .noneMatch(that::overlaps);
+                }
+            }
+            if (!cleared)
+            {
+                return false;
+            }
+        }
+        return true;
+    }
+
+    @Override
+    public boolean fullyGeometricallyEncloses(final Segment segment)
```

Comment:
Something we discussed offline: could we potentially leverage this method when looking at `fullyGeometricallyEncloses(final PolyLine polyline)`? That might need more testing.

Reply:
Did some performance testing and this is not as performant so I am going to remove and remove from the interface.